galleries/examples/ticks/tick-locators.py

```diff
@@ -53,7 +53,7 @@ def setup(ax, title):
 
 # Index Locator
 setup(axs[4], title="IndexLocator(base=0.5, offset=0.25)")
-axs[4].plot(range(0, 5), [0]*5, color='white')
+axs[4].plot([0]*5, color='white')
```

Comment:
It's not completely obvious from looking at this example what the difference is between `MultipleLocator` and `IndexLocator`.  But from the [API docs](https://matplotlib.org/devdocs/api/ticker_api.html#matplotlib.ticker.IndexLocator) it seems to me that `IndexLocator` makes most sense when the axis does not have numbers associated with it.  So I took out the x-values.